panos_upgrade_assurance/check_firewall.py

```diff
@@ -1253,6 +1257,37 @@ def run_health_checks(
 
         return result
 
+    @staticmethod
+    def check_version_against_version_match_dict(version: Version, match_dict: dict):
+        """Compare the given software version against the match dict.
+
+        # Parameters
+        version (str): The software version to compare. Example: "10.1.11"
+        match_dict (dict): A dictionary of tuples mapping major/minor versions to match criteria
+            example
+
+            ```python
+            {
+                "81": [("==", "8.1.21.2"), (">=", "8.1.25.1")],
+                "90": [(">=", "9.0.16.5")],
+            }
+            ```
+
+        Returns
```

Comment:
update docstring for pan.dev
```suggestion
        # Returns
```